addons/context_actions/CfgContext.hpp

```diff
@@ -343,6 +343,11 @@ class EGVAR(context_menu,actions) {
             statement = QUOTE(_objects call FUNC(refuelVehicles));
             icon = "\a3\ui_f\data\igui\cfg\simpleTasks\types\refuel_ca.paa";
         };
+        class SwitchWeapon {
+            displayName = "$STR_A3_Switch1";
+            icon = "\a3\ui_f\data\GUI\Cfg\Hints\VehicleAmmo_CA.paa";
+            insertChildren = QUOTE(call FUNC(getVehicleWeaponActions));
```

Comment:
Runtimes on a tank:
`canSelectVehicleWeapon` 0.0463 ms
`getVehicleWeaponActions` 1.9666 ms
Is it better to keep the quicker condition check function:
`condition = QUOTE(_hoveredEntity call FUNC(canSelectVehicleWeapon));`
or is the difference small enough on a 1-run-per-context-open that it should just depend on the child actions?